src/utils/buildButtons.ts

```diff
@@ -0,0 +1,15 @@
+import { MessageActionRow, MessageButton } from "discord.js";
+
+export function buildButtons(choices: string[]): MessageActionRow[] {
```

Comment:
the function and file should have a more descriptive name like perhaps `buildTriviaButtons` as `buildButtons` is rather vague and the bot will be dealing with building a lot of other buttons in the future (ie for more consistency).